features/api_documentation.feature

```diff
@@ -14,18 +17,21 @@ Scenario: Sign in page for NOT API sandbox
      When I click on the API Sign up and Documentation link
      Then I should be directed to the API landing page
 
+  @advocate
```

Comment:
maybe we should have a @vendor or @api tag since advocates, per se, are not the target of this specific feature